### ISSUE TITLE
Add additional cursor commands

### ISSUE THREAD

**Issue by zrzka** [crossterm-rs/crossterm#298]
While experimenting with `anes`, found more useful cursor commands like:

* `MoveCursorToNextLine`
  * Moves cursor to the next line (or `n` lines) and then moves the cursor to the first column
* `MoveCursorToPreviousLine`
  * Moves cursor to the previous line (or `n` lines) and then moves the cursor to the first column
* `MoveCursorToColumn`
  * Moves cursor to the given column on the current row

https://github.com/zrzka/anes-rs/blob/4a7fe5a601d589977eec201c710e382dfe016e79/anes/src/cursor.rs#L67-L87

These commands allows to make nicer UI in an easier way:

```rust
for line in MENU.split('\n') {
    queue!(w, line, anes::MoveCursorToNextLine(1))?;
}
```

Compare with the current `stderr.rs` example (`mut y` & `Goto`):

```rust
let mut y = 1;
for line in TEXT.split('\n') {
    queue!(w, Goto(1, y))?;
    queue!(w, Output(line.to_string()))?;
    y += 1;
}
```

I tested them on Linux/macOS. But have no idea if they do work properly on the Windows. Looks like a nice addition.

**Comment by TimonPost** [comment]
That is great. It sounds like a nice addition. We can simulate this with WinApi quite easy as well. It will probably work about the same as clear.

**Comment by TimonPost** [comment]
Closing fixed in 0.14
